src/main.c

```diff
@@ -315,6 +332,21 @@ char simulator_tone[5];
 char qsos[MAX_QSOS][LOGLINELEN+1];
 int nr_qsos = 0;
 
+int qsoflags_for_qtc[MAX_QSOS];
+int nr_qsosflags_for_qtc;
+int next_qtc_qso;
+t_qtclist qtclist;
+int nr_qtcsent = 0;
+t_qtcreclist qtcreclist;
+struct t_qtc_store_obj *qtc_temp_obj;
+int qtcdirection = 0;
+//char qtc_ry_lines[12][50] = {"", "", "", "", "", "", "", "", "", "", "", ""};
+//int qtc_ry_lines_attr[12] = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
+t_qtc_ry_line qtc_ry_lines[QTC_RY_LINE_NR];
+int qtc_ry_currline = 0;
+int qtc_ry_capture;
+int qtc_ry_copied;
+
```

Comment:
Done, see the commit: https://github.com/airween/tlf/commit/9d32cee8fcc0452d0237ea6370c8b7cc5f5aad9e